packages/sdks/web-component/src/lib/descope-wc/DescopeWc.ts

```diff
@@ -229,7 +229,12 @@ class DescopeWc extends BaseDescopeWc {
           moduleRes?.start?.();
           return moduleRes;
         }
-        const module = await loadSdkScript(script.id);
+        await this.injectNpmLib(
+          '@descope/flow-scripts',
+          'latest',
```

Comment:
I think this is very dangerous
Maybe we should consider a different strategy

Once we will release a new version of our scripts, all customers will get it in production
In case something is broken, their prod env will be down

Reply:
I think our approach should be: 
you didn’t change anything - everything remains the same
No unexpected changes

What I can think of right now is setting a fixed version for the scripts and updating it with SDK releases,
just like any other standard dependency

WDYT?

Reply:
that would defeat the purpose of this pr tho, as customers will have to update sdks for new scripts. i think its easer to mitigate the problem in our devops infra rather then forcing customers to use a fixed version